### DISCUSSION TITLE
Format settings for project numbers

### DISCUSSION THREAD

**Discussion by xx1982xx38** [kimai/kimai#5163] [General]
Hello, I would like my project to have the following when I create a project.

The project is originally created with a 4-digit number "0002" etc. I would now like it to say "24-0002"

I saw that you can change the format in the settings, but I can't find what value needs to be entered for this format to be displayed.

I hope you can help me with your collective knowledge.

Thanks

**Comment by kevinpapst** [comment]
<img width="515" alt="Bildschirmfoto 2024-11-19 um 15 08 06" src="https://github.com/user-attachments/assets/c9cbbf9b-cc77-4800-8aae-79685158052a">
